clock/src/main/java/io/deephaven/base/clock/Clock.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.base.clock;
+
+import java.time.Instant;
+
+/**
+ * Provides time-based methods. Callers should prefer the methods most appropriate for their use-case and desired
+ * resolution. The performance of the methods may differ depending on the resolution they provide.
+ *
+ * @see SystemClock
+ */
+public interface Clock {
+
+    /**
+     * The {@link SystemClock}. Provides singleton semantics around {@link SystemClock#of()}.
+     *
+     * @return the system clock
+     */
+    static SystemClock systemUTC() {
+        return SystemClockInstance.INSTANCE;
+    }
```

Comment:
I don't think this method belongs.  It is specific to a single clock, and that clock can be obtained via other methods.

Reply:
I think the method may actually be better named `system()`...

That said, it does serve an important service not fulfilled by other methods - it's essentially providing a singleton cache around `SystemClock.of()`. (`SystemClockInstance` itself is not public.)